components/AppHeader.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <CHeader class="header-sticky p-0" :class="{'shadow-sm': isScrolled}">
+    <CContainer fluid class="border-bottom px-3 mt-1 justify-content-start">
+      <CHeaderToggler @click="toggleSidebar">
+        <span data-testid="toggle-sidebar-button">
+          <CIcon icon="cilMenu" size="lg"/>
+        </span>
+      </CHeaderToggler>
+      <CHeaderBrand href="/">
+        <CIcon icon="cilGlobeAlt" size="lg"/>
+        <span class="ms-2 py">DAEDALUS Explore</span>
+      </CHeaderBrand>
+      <div v-show="showBreadcrumbs" class="ms-5 d-none d-xxl-block">
+        <BreadCrumb/>
+      </div>
+      <CHeaderNav class="ms-auto d-xxl-none">
+        <CNavItem class="py-1">
+          <VerticalRule/>
+        </CNavItem>
+        <CNavItem href="#">
+          <CIcon icon="cilCloudDownload" size="lg"/>
+        </CNavItem>
+        <CNavItem class="py-1">
+          <VerticalRule/>
+        </CNavItem>
+        <CNavItem href="#"> ?</CNavItem>
+      </CHeaderNav>
+    </CContainer>
+    <CContainer v-show="showBreadcrumbs" fluid class="d-xxl-none full-breadcrumb-container">
+      <BreadCrumb/>
+    </CContainer>
+  </CHeader>
+</template>
+
+<script setup lang="ts">
+import { CIcon } from '@coreui/icons-vue';
+import throttle from 'lodash.throttle';
+import { useRoute } from 'vue-router';
+
+const route = useRoute();
```

Comment:
just a tiny layout thing but may be best to seperate your variables at the top and functions below so there is a clear distinction

Reply:
Hopefully this will be sorted out by https://github.com/j-idea/daedalus-web-app/pull/2#issuecomment-2225919107